.github/workflows/test-download.yaml

```
@@ -0,0 +1,52 @@
+name: Tests (Download)
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  tests:
+
+    strategy:
+      fail-fast: false  # continue other tests if one test in matrix fails
+      matrix:
+        node-version: [16.x]
+        os: [ubuntu-latest]
+
+    name: Test kit on Node v${{ matrix.node-version }} (${{ matrix.os }})
```

Can we give this test a different name so we can differentiate it from the other tests?

Something like 'Test downloading kit from website' maybe?

```suggestion
    name: Test downloading kit from website
```